virt-v2v/cold/scripts/rhel/run/network_config_util.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
```

Comment:
I see this script gettign more and more complex but don't see much logs, it might be hard for the developers to debug in future.
Bit hacky, but this might work as a log from the complex functions:
```
# New FD for log
exec 3>&1
log() {
    echo $@ >&3
}
```
or we could use the basic stderr &2

Reply:
cool, added logs :books: